### PR TITLE
Replace .NET Framework 3.5 tokens

### PR DESCRIPTION
**Pull request by nxtn**

### CODE REVIEW

File: docs/framework/wcf/feature-details/working-with-certificates.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ With this capability enabled, you can set the <xref:System.ServiceModel.Security
 
 Mapping an X.509 certificate to the token that represents a Windows user account is considered an elevation of privilege because, once mapped, the Windows token can be used to gain access to protected resources. Therefore, domain policy requires the X.509 certificate to comply with its policy prior to mapping. The *SChannel* security package enforces this requirement.
 
-When using [!INCLUDE[netfx35_long](../../../../includes/netfx35-long-md.md)] or later, WCF ensures the certificate conforms to domain policy before it is mapped to a Windows account.
+When using .NET Framework version 3.5 or later, WCF ensures the certificate conforms to domain policy before it is mapped to a Windows account.
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/working-with-certificates.md]:
```suggestion
When using .NET Framework 3.5 or later versions, WCF ensures the certificate conforms to domain policy before it is mapped to a Windows account.
```

File: docs/framework/wcf/host-a-wcf-service-net-framework-3-5-iis--net-framework-4.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ ms.date: "03/30/2017"
 ms.assetid: 9aabc785-068d-4d32-8841-3ef39308d8d6
 ---
 # How to: Host a WCF Service Written with .NET Framework 3.5 in IIS Running Under .NET Framework 4
-When hosting a Windows Communication Foundation (WCF) service written with [!INCLUDE[netfx35_long](../../../includes/netfx35-long-md.md)] on a machine running [!INCLUDE[netfx40_long](../../../includes/netfx40-long-md.md)], you may get a <xref:System.ServiceModel.ProtocolException> with the following text.  
+When hosting a Windows Communication Foundation (WCF) service written with .NET Framework version 3.5 on a machine running [!INCLUDE[netfx40_long](../../../includes/netfx40-long-md.md)], you may get a <xref:System.ServiceModel.ProtocolException> with the following text.
```
Comment (mairaw) [inline docs/framework/wcf/host-a-wcf-service-net-framework-3-5-iis--net-framework-4.md]:
```suggestion
When hosting a Windows Communication Foundation (WCF) service written with .NET Framework 3.5 on a machine running [!INCLUDE[netfx40_long](../../../includes/netfx40-long-md.md)], you may get a <xref:System.ServiceModel.ProtocolException> with the following text.  
```

File: docs/framework/wcf/synchronous-and-asynchronous-operations.md [hunk 3]
```diff
@@ -168,7 +168,7 @@ svcutil http://localhost:8000/servicemodelsamples/service/mex /async /tcv:Versio
...
+ The event-based asynchronous model, however, is only available in .NET Framework version 3.5. In addition, it is not supported even in .NET Framework 3.5 when a WCF client channel is created by using a <xref:System.ServiceModel.ChannelFactory%601?displayProperty=nameWithType>. With WCF client channel objects, you must use <xref:System.IAsyncResult?displayProperty=nameWithType> objects to invoke your operations asynchronously. To use this approach, specify the **/async** command option with the [ServiceModel Metadata Utility Tool (Svcutil.exe)](servicemodel-metadata-utility-tool-svcutil-exe.md), as in the following example.
```
Comment (mairaw) [inline docs/framework/wcf/synchronous-and-asynchronous-operations.md]:
```suggestion
 The event-based asynchronous model, however, is only available in .NET Framework 3.5. In addition, it is not supported even in .NET Framework 3.5 when a WCF client channel is created by using a <xref:System.ServiceModel.ChannelFactory%601?displayProperty=nameWithType>. With WCF client channel objects, you must use <xref:System.IAsyncResult?displayProperty=nameWithType> objects to invoke your operations asynchronously. To use this approach, specify the **/async** command option with the [ServiceModel Metadata Utility Tool (Svcutil.exe)](servicemodel-metadata-utility-tool-svcutil-exe.md), as in the following example.  
```

File: docs/framework/windows-workflow-foundation/pick-activity.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ ms.date: "03/30/2017"
...
+The <xref:System.Activities.Statements.Pick> activity simplifies the modeling of a set of event triggers followed by their corresponding handlers.  A <xref:System.Activities.Statements.Pick> activity contains a collection of <xref:System.Activities.Statements.PickBranch> activities, where each <xref:System.Activities.Statements.PickBranch> is a pairing between a <xref:System.Activities.Statements.PickBranch.Trigger%2A> activity and an <xref:System.Activities.Statements.PickBranch.Action%2A> activity.  At execution time, the triggers for all branches are executed in parallel.  When one trigger completes, then its corresponding action is executed, and all other triggers are canceled.  The behavior of the [!INCLUDE[netfx_current_short](../../../includes/netfx-current-short-md.md)]<xref:System.Activities.Statements.Pick> activity is similar to the .NET Framework 3.5<xref:System.Workflow.Activities.ListenActivity> activity.
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/pick-activity.md]: added a missing space

```suggestion
The <xref:System.Activities.Statements.Pick> activity simplifies the modeling of a set of event triggers followed by their corresponding handlers.  A <xref:System.Activities.Statements.Pick> activity contains a collection of <xref:System.Activities.Statements.PickBranch> activities, where each <xref:System.Activities.Statements.PickBranch> is a pairing between a <xref:System.Activities.Statements.PickBranch.Trigger%2A> activity and an <xref:System.Activities.Statements.PickBranch.Action%2A> activity.  At execution time, the triggers for all branches are executed in parallel.  When one trigger completes, then its corresponding action is executed, and all other triggers are canceled.  The behavior of the [!INCLUDE[netfx_current_short](../../../includes/netfx-current-short-md.md)]<xref:System.Activities.Statements.Pick> activity is similar to the .NET Framework 3.5 <xref:System.Workflow.Activities.ListenActivity> activity.  
```

File: docs/framework/windows-workflow-foundation/samples/activity-library.md [hunk 5]
```diff
@@ -18,7 +18,7 @@ This section contains samples that demonstrate advanced custom activities in Win
  Demonstrates how to create activities that allow accessing databases to retrieve or modify information and use [ADO.NET](https://go.microsoft.com/fwlink/?LinkId=166081) to access the database.  
   
  [Externalized Policy Activity in .NET Framework 4.5](externalized-policy-activity-in-net-framework-4-5.md)  
- Demonstrates how the ExternalizedPolicy4 activity allows executing existing Windows Workflow Foundation in [!INCLUDE[netfx35_long](../../../../includes/netfx35-long-md.md)] (WF 3.5) <xref:System.Workflow.Activities.Rules.RuleSet> objects in Windows Workflow Foundation in [!INCLUDE[netfx_current_long](../../../../includes/netfx-current-long-md.md)] (WF 4.5) directly by using the rules engine that is shipped in WF 3.5. 
+ Demonstrates how the ExternalizedPolicy4 activity allows executing existing Windows Workflow Foundation in .NET Framework version 3.5 (WF 3.5) <xref:System.Workflow.Activities.Rules.RuleSet> objects in Windows Workflow Foundation in [!INCLUDE[netfx_current_long](../../../../includes/netfx-current-long-md.md)] (WF 4.5) directly by using the rules engine that is shipped in WF 3.5.
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/samples/activity-library.md]:
```suggestion
 Demonstrates how the ExternalizedPolicy4 activity allows executing existing Windows Workflow Foundation in .NET Framework 3.5 (WF 3.5) <xref:System.Workflow.Activities.Rules.RuleSet> objects in Windows Workflow Foundation in [!INCLUDE[netfx_current_long](../../../../includes/netfx-current-long-md.md)] (WF 4.5) directly by using the rules engine that is shipped in WF 3.5. 
```

File: docs/framework/windows-workflow-foundation/samples/externalized-policy-activity-in-net-framework-4-5.md [hunk 6]
```diff
@@ -5,7 +5,7 @@ ms.assetid: 92fd6f92-23a1-4adf-b96a-2754ea93ad3e
...
+This sample demonstrates how the ExternalizedPolicy4 activity allows executing existing .NET Framework version 3.5 Windows Workflow Foundation (WF 3.5) <xref:System.Workflow.Activities.Rules.RuleSet> objects in [!INCLUDE[netfx_current_long](../../../../includes/netfx-current-long-md.md)] Windows Workflow Foundation (WF 4.5) directly by using the rules engine that is shipped in WF 3.5. By using this activity, you can open and execute any existing WF 3.5 <xref:System.Workflow.Activities.Rules.RuleSet>. For more information about WF 3.5 Rules Engine included as part of Windows Workflow Foundation, please read [Introduction to the Windows Workflow Foundation Rules Engine](https://go.microsoft.com/fwlink/?LinkId=166079). For more information about migrating rules to [!INCLUDE[wf1](../../../../includes/wf1-md.md)] in [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)], please read the migration guidance at [Migration Guidance](../migration-guidance.md).
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/samples/externalized-policy-activity-in-net-framework-4-5.md]:
```suggestion
This sample demonstrates how the ExternalizedPolicy4 activity allows executing existing .NET Framework 3.5 Windows Workflow Foundation (WF 3.5) <xref:System.Workflow.Activities.Rules.RuleSet> objects in [!INCLUDE[netfx_current_long](../../../../includes/netfx-current-long-md.md)] Windows Workflow Foundation (WF 4.5) directly by using the rules engine that is shipped in WF 3.5. By using this activity, you can open and execute any existing WF 3.5 <xref:System.Workflow.Activities.Rules.RuleSet>. For more information about WF 3.5 Rules Engine included as part of Windows Workflow Foundation, please read [Introduction to the Windows Workflow Foundation Rules Engine](https://go.microsoft.com/fwlink/?LinkId=166079). For more information about migrating rules to [!INCLUDE[wf1](../../../../includes/wf1-md.md)] in [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)], see the [Migration Guidance](../migration-guidance.md).
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @NextTurn. Left a couple of comments for you to consider.

File: docs/framework/wcf/feature-details/how-to-use-a-custom-user-name-and-password-validator.md [hunk 7]
```diff
@@ -39,7 +39,7 @@ For a sample application, see [User Name Password Validator](../samples/user-nam
     When using transport-level security over HTTP(S), add either the [\<wsHttpBinding>](../../configure-apps/file-schema/wcf/wshttpbinding.md) or [\<basicHttpBinding>](../../configure-apps/file-schema/wcf/basichttpbinding.md), a [\<netTcpBinding>](../../configure-apps/file-schema/wcf/nettcpbinding.md) or a [\<customBinding>](../../configure-apps/file-schema/wcf/custombinding.md) that uses HTTP(S) and the `Basic` authentication scheme.
 
     > [!NOTE]
-    > When [!INCLUDE[netfx35_long](../../../../includes/netfx35-long-md.md)] or later is used, you can use a custom username and password validator with message and transport security. With WinFX, a custom username and password validator can only be used with message security.
+    > When .NET Framework version 3.5 or later is used, you can use a custom username and password validator with message and transport security. With WinFX, a custom username and password validator can only be used with message security.
```
Comment (NextTurn) [inline docs/framework/wcf/feature-details/how-to-use-a-custom-user-name-and-password-validator.md]:
```suggestion
    > When .NET Framework 3.5 or later version is used, you can use a custom username and password validator with message and transport security. With WinFX, a custom username and password validator can only be used with message security.
```

Any suggestion for the "version X or later" pattern?

File: docs/framework/wcf/feature-details/elevation-of-privilege.md [hunk 8]
```diff
@@ -32,7 +32,7 @@ ms.assetid: 146e1c66-2a76-4ed3-98a5-fd77851a06d9
  If you want to have deterministic behavior when using Windows authentication together with impersonation you need to explicitly set the Windows credential or you need to establish a security context with the service. To do this, use a message security session or a transport security session. For example, the net.tcp transport can provide a transport security session. Additionally, you must use only a synchronous version of client operations when calling the service. If you establish a message security context, you should not keep the connection to the service open longer than the configured session renewal period, because the identity can also change during the session renewal process.  
   
 ### Credentials Capture  
- The following applies to [!INCLUDE[netfx35_long](../../../../includes/netfx35-long-md.md)], and subsequent versions.  
+ The following applies to .NET Framework version 3.5, and subsequent versions.
```
Comment (NextTurn) [inline docs/framework/wcf/feature-details/elevation-of-privilege.md]:
```suggestion
 The following applies to .NET Framework 3.5, and subsequent versions.  
```

Shall I remove other "version" strings?

File: docs/framework/wcf/feature-details/how-to-use-a-custom-user-name-and-password-validator.md (hunk 7 above)
Comment (mairaw) [reply docs/framework/wcf/feature-details/how-to-use-a-custom-user-name-and-password-validator.md]: We usually say .NET Framework 3.5 or later versions

File: docs/framework/wcf/feature-details/elevation-of-privilege.md (hunk 8 above)
Comment (mairaw) [reply docs/framework/wcf/feature-details/elevation-of-privilege.md]: Yep, if it's related to the changes you're making

File: docs/framework/wcf/feature-details/partial-trust.md [hunk 9]
```diff
@@ -4,7 +4,7 @@ ms.date: "03/30/2017"
 ms.assetid: 489b1587-9909-4d0e-8c1a-5e83c8f8292b
 ---
 # Partial Trust
-Starting with the [!INCLUDE[netfx35_long](../../../../includes/netfx35-long-md.md)], partially trusted callers can access public types and methods implemented in <xref:System.ServiceModel>, <xref:System.Runtime.Serialization>, and <xref:System.ServiceModel.Web>. This section describes supported scenarios for using Windows Communication Foundation (WCF) within a partially trusted application as well as the limited subset of WCF functionality available to applications running with reduced code access security (CAS) permissions.  
+Starting with the .NET Framework version 3.5, partially trusted callers can access public types and methods implemented in <xref:System.ServiceModel>, <xref:System.Runtime.Serialization>, and <xref:System.ServiceModel.Web>. This section describes supported scenarios for using Windows Communication Foundation (WCF) within a partially trusted application as well as the limited subset of WCF functionality available to applications running with reduced code access security (CAS) permissions.
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/partial-trust.md]:
```suggestion

Starting with the .NET Framework 3.5, partially trusted callers can access public types and methods implemented in <xref:System.ServiceModel>, <xref:System.Runtime.Serialization>, and <xref:System.ServiceModel.Web>. This section describes supported scenarios for using Windows Communication Foundation (WCF) within a partially trusted application as well as the limited subset of WCF functionality available to applications running with reduced code access security (CAS) permissions.  
```

File: docs/framework/wcf/feature-details/reliable-messaging-protocol-version-1-1.md [hunk 10]
```diff
@@ -5,7 +5,7 @@ ms.assetid: 0da47b82-f8eb-42da-8bfe-e56ce7ba6f59
 ---
 # Reliable Messaging Protocol version 1.1
 
-This topic covers Windows Communication Foundation (WCF) implementation details for the WS-ReliableMessaging February 2007 (version 1.1) protocol necessary for interoperation using the HTTP transport. WCF follows the WS-ReliableMessaging specification with the constraints and clarifications explained in this topic. Note that the WS-ReliableMessaging version 1.1 protocol is implemented starting with the [!INCLUDE[netfx35_long](../../../../includes/netfx35-long-md.md)].
+This topic covers Windows Communication Foundation (WCF) implementation details for the WS-ReliableMessaging February 2007 (version 1.1) protocol necessary for interoperation using the HTTP transport. WCF follows the WS-ReliableMessaging specification with the constraints and clarifications explained in this topic. Note that the WS-ReliableMessaging version 1.1 protocol is implemented starting with the .NET Framework version 3.5.
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/reliable-messaging-protocol-version-1-1.md]:
```suggestion
This topic covers Windows Communication Foundation (WCF) implementation details for the WS-ReliableMessaging February 2007 (version 1.1) protocol necessary for interoperation using the HTTP transport. WCF follows the WS-ReliableMessaging specification with the constraints and clarifications explained in this topic. Note that the WS-ReliableMessaging version 1.1 protocol is implemented starting with .NET Framework 3.5.
```

**Comment by mairaw** [comment]
Now that I merged your other PR, there are some merge conflicts here

**Comment by NextTurn** [comment]
I noticed that, but there is no way to pre-resolve the conflicts before they occured :smile:.

**Comment by mairaw** [comment]
Yeah, I know 😄 
I've fixed the merge conflicts.

File: docs/framework/data/wcf/data-service-versioning-wcf-data-services.md [hunk 11]
```diff
@@ -58,8 +58,8 @@ The Open Data Protocol (OData) enables you to create data services so that clien
 
 |OData Protocol Version|Support introduced in…|
 |-----------------------------------------------------------------------------------|----------------------------|
-|Version 1|-   [!INCLUDE[netfx35_long](../../../../includes/netfx35-long-md.md)] Service Pack 1 (SP1)<br />-   Silverlight version 3|
-|Version 2|-   .NET Framework 4<br />-   An update to [!INCLUDE[netfx35_long](../../../../includes/netfx35-long-md.md)] SP1. You can download and install the update from the [Microsoft Download Center](https://go.microsoft.com/fwlink/?LinkId=158125).<br />-   Silverlight version 4|
+|Version 1|-   .NET Framework version 3.5 Service Pack 1 (SP1)<br />-   Silverlight version 3|
```
Comment (mairaw) [inline docs/framework/data/wcf/data-service-versioning-wcf-data-services.md]:
```suggestion
|Version 1|-   .NET Framework 3.5 Service Pack 1 (SP1)<br />-   Silverlight version 3|
```

**Review by mairaw — APPROVED** [review]
Thank you @NextTurn!

**Pull request merged**